wagtailio/project_styleguide/templates/patterns/includes/menu_footer.html

```diff
@@ -0,0 +1,30 @@
+{% if footer_menu_sections %}
+    <nav class="footer-menu" aria-label="Main menu">
```

Comment:
I had this initially then read about [repeated landmarks](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#repeated_landmarks) as I thought this menu would be a repeat of the header menu... however thinking about it again it won't be identical as the main menu has the large CTAs, so this would be better marks up as the footer menu